ui/appui-react/src/appui-react/layout/state/internal/TabStateHelpers.ts

```diff
@@ -53,6 +53,7 @@ export function createDraggedTabState(
       widgetIndex: 0,
     },
     position: { x: 0, y: 0 },
+    active: true,
```

Comment:
Is this necessary? Looks like `createDraggedTabState` is always called with `active` in `args`.

Reply:
Good point, I moved it to test utils.